packages/domains-table/src/domains-table/style.scss

```diff
@@ -457,4 +457,9 @@ main.domains-overview main.domains-overview__list .multi-sites-dashboard-layout-
 		padding: 16px 24px;
 		margin-inline-start: 0;
 	}
+
+	.domains-table__row.is-selected > td {
+		background-color: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+		color: #1e1e1e;
```

Comment:
We can probably use a color variable: `--theme-base-color` here.

Reply:
Thanks! Updated here https://github.com/Automattic/wp-calypso/pull/97619/commits/f800c4d759138b169b091bd663594415ab8a8dd9